_includes/devdoc/guide_dash_features.md

```diff
@@ -878,3 +878,61 @@ Note the following timeouts defined by Dash Core related to signing sessions:
 | MN Broadcast | 10      | Majority | _Deprecated by DIP3 (deterministic masternode list) in Dash Core 0.13._<br><br>If a majority (6+) of nodes agree, a new `mnb` message is not required.
 
 {% endautocrossref %}
+
+### Proof of Service
+{% include helpers/subhead-links.md %}
+
+{% autocrossref %}
+
+The Proof of Service (PoSe) scoring system helps incentivize masternodes to
+provide network services. Masternodes that neglect to participate receive an
+increased PoSe score which eventually results in them being excluded from
+masternode payment eligibility.
+
+The current PoSe scoring system is based on participation in the LLMQ DKG
+process. This scoring system will expand over time to incorporate additional
+service requirements in support of the future Dash Platform (Evolution)
+functionality.
+
+| Service | Percent of Score | Requirement |
+| ----------- | ---- | ------------------- |
+| LLMQ DKG    | 100% | Participate in the DKG process used to establish LLMQs. Requires exchanging messages with other quorum members. |
+
+**PoSe Score Calculation**
+
+As shown in the following table, the PoSe Score always decreases by 1 per block
+as long as a masternode has not been banned. Once banned, the masternode can
+only be restored by sending a Provider Update Registrar (ProUpRegTx) special
+transaction.
```

Comment:
Don't remember this being the case. I thought they auto-unbanned

Reply:
Good catch. They don't auto-unban, but I have the wrong special tx (it's [ProUpServTx](https://github.com/dashpay/dips/blob/master/dip-0003.md#updating-service-features-from-operator-proupservtx))